view/frontend/web/instantsearch.js

```diff
@@ -96,6 +92,92 @@ requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBu
 					$('.algolia-instant-replaced-content').hide();
 					$('.algolia-instant-selector-results').show();
 				}
+			},
+			// The url is now rendered as follows : http://website.com?q=searchquery&facet1=value&facet2=value1~value2
+			// "?" and "&" are used to be fetched easily inside Magento for the backend rendering
+			// Multivalued facets use "~" as separator
+			// Targeted index is defined by sortBy parameter
+			routing : {
```

Comment:
Nice job! 🚀 
Can the `routing` parameter be defined on a different place of the extension e.g, common.js file?
It's only for a readability reasons as the routing definition is pretty long and I think it'll be easier to read it if it's extracted to separated method and only this new method is called here. WDYT?